### PR TITLE
Test maps only

### PR DESCRIPTION
**Pull request by vramik**

### CODE REVIEW

File: model/map/src/main/java/org/keycloak/models/map/client/MapClientProvider.java [hunk 1]
```diff
@@ -245,6 +245,13 @@ public ClientModel getClientById(String id, RealmModel realm) {
             return null;
         }
 
+        //TODO: replace with Pattern/Matcher or find a better place to check if clientUuid is in uuid format
```
Comment (vramik) [inline model/map/src/main/java/org/keycloak/models/map/client/MapClientProvider.java]: the problem is that from rest services can come request with id in non-uuid format and it throws IllegalArgumentException.

Comment (hmlnarik) [reply model/map/src/main/java/org/keycloak/models/map/client/MapClientProvider.java]: This introduces silent ignoring of an error. Throwing an error (L 257) should be thus kept.

File: model/map/src/main/java/org/keycloak/models/map/client/MapClientEntity.java [hunk 2]
```diff
@@ -415,7 +415,7 @@ public void removeAuthenticationFlowBindingOverride(String binding) {
     }
 
     public void setAuthenticationFlowBindingOverride(String binding, String flowId) {
-        this.updated |= ! Objects.equals(this.nodeReRegistrationTimeout, nodeReRegistrationTimeout);
+        this.updated |= Objects.equals(this.nodeReRegistrationTimeout, nodeReRegistrationTimeout);
```
Comment (vramik) [inline model/map/src/main/java/org/keycloak/models/map/client/MapClientEntity.java]: This one fixes org.keycloak.testsuite.cli.KcinitTest.testBrowserContinueAuthenticator

Comment (hmlnarik) [reply model/map/src/main/java/org/keycloak/models/map/client/MapClientEntity.java]: Isn't this line (even in the original) utterly wrong? The `equals` should be on the parameters, not on nodeReRegistrationTimeout. Could you please check also other setters for similar bug?

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**